scripts/send-test-event.sh

```diff
@@ -6,15 +6,14 @@ source ${ROOT_DIR}/scripts/env-local.sh
 export CURL_CA_BUNDLE="${ROOT_DIR}/ssl/data/ca.pem"
 
 curl -v \
---header "Content-Type: application/json" \
+--header "Content-Type: application/csv" \
```

Comment:
Create two test scripts. One to write a JSON event (like the original) and another to send a CSV.

Reply:
created a new script for csv